examples/sparse_2of4_quantization_fp8/README.md

```diff
@@ -106,6 +106,17 @@ Output Directories:
 - Without FP8: `Meta-Llama-3-8B-Instruct-2of4-sparse`
 - With FP8: `Meta-Llama-3-8B-Instruct-2of4-W8A8-FP8-Dynamic-Per-Token`
 
+#### Saving Without Sparse Compression
+
+To save the model on disk without sparse compression:
+
+```python
+model.save_pretrained(save_dir, save_compressed=True, no_sparse_compression=True)
```

Comment:
In the future, can we rename this to something like `ignore_sparse_compression`?

Reply:
`disable_sparse_compression`? credits @tlrmchlsmth

Reply:
Resolving this for now